### PR TITLE
gBoards BuzzSaw 

### PR DESCRIPTION
**Pull request by germ**
Initial Commit adding support for the BuzzSaw from g Heavy Industries

- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

### CODE REVIEW

**Comment by ridingqwerty** [comment]
`File changed: 33`

I’m thinking you didn’t want to sneak in the Faunchpad, Gergoplex, and Virtser changes with this PR.

**Comment by germ** [comment]
@ridingqwerty oh god you're right. Give me a few

**Comment by germ** [comment]
Git is hard, but at least I know how to use checkouts and whatnot now. Should be good @ridingqwerty

**Comment by ridingqwerty** [comment]
It’s all good @germ I suck at git

**Comment by germ** [comment]
At least there's no forced pushes in this PR (Unlike in GergoPlex and co lol)

**Comment by drashna** [comment]
... don't look at my PRs then.  Force pushes all over the place. :D

File: keyboards/buzzsaw/keymaps/default/rules.mk [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+VERBOSE 		 = yes
+DEBUG_MATRIX_SCAN_RATE   = no
+DEBUG_BALLER 		 = no
+DEBUG_MATRIX		 = yes
+
+# A bunch of stuff that you shouldn't touch unless you
+# know what you're doing.
+#
+# No touchy, capiche?
+SRC += matrix.c i2c_master.c
```
Comment (drashna) [inline keyboards/buzzsaw/keymaps/default/rules.mk]: This shouldn't be here. It should be in the keyboard's rules.mk, which ... it is. 
```suggestion
```

File: keyboards/buzzsaw/readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# BuzzSaw
+
+![BuzzSaw](https://assets.bigcartel.com/product_images/247417775/IMG_20191025_1410027-01.jpeg)
+
+A Alice style Choc only keyboard that is breakable down to a 40% Split semi-ergo
+
+[More info on gboards.ca](http://gboards.ca)
+
+Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)
```
Comment (drashna) [inline keyboards/buzzsaw/readme.md]:
```suggestion
* Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
```

File: keyboards/buzzsaw/readme.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# BuzzSaw
+
+![BuzzSaw](https://assets.bigcartel.com/product_images/247417775/IMG_20191025_1410027-01.jpeg)
+
+A Alice style Choc only keyboard that is breakable down to a 40% Split semi-ergo
+
+[More info on gboards.ca](http://gboards.ca)
+
+Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
+Hardware Supported: Gergo (Kit, Partial, Ready)
```
Comment (drashna) [inline keyboards/buzzsaw/readme.md]:
```suggestion
* Hardware Supported: Gergo (Kit, Partial, Ready)   
```

File: keyboards/buzzsaw/readme.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+![BuzzSaw](https://assets.bigcartel.com/product_images/247417775/IMG_20191025_1410027-01.jpeg)
+
+A Alice style Choc only keyboard that is breakable down to a 40% Split semi-ergo
+
+[More info on gboards.ca](http://gboards.ca)
+
+Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
+Hardware Supported: Gergo (Kit, Partial, Ready)   
+Hardware Availability: [gboards.ca](http://gboards.ca)
```
Comment (drashna) [inline keyboards/buzzsaw/readme.md]:
```suggestion
* Hardware Availability: [gboards.ca](http://gboards.ca)  
```

File: keyboards/buzzsaw/buzzsaw.h [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+#define OLATB           0x15
+
+/*  ---------- LEFT HAND -----------   ---------- RIGHT HAND ---------- */
+#define LAYOUT_buzzsaw(                                                    							\
+    N00,N01,N02,N03,N04,N05,N06,         N07,N08,N09,N10,N11,N12,N13,N14,			  A01,		\
+    L00,L01,L02,L03,L04,L05,             R00,R01,R02,R03,R04,R05,R06,R07,   	  A02, 	  \
+    L10,L11,L12,L13,L14,L15,     	  		 		 R10,R11,R12,R13,R14,R15,R16, 	   	A03, 	  \
+    L20,L21,L22,L23,L24,L25,   				 				   R20,R21,R22,R23,R24,R25,	   		K00, 	  \
+    L30,L31,L32,L33,   									 								 		     R30,R31,   K01,K02,K03)\
+{                               			 		 \
```
[the remark this answers is not in the document]
Comment (germ) [reply keyboards/buzzsaw/buzzsaw.h]: retabbed!

File: keyboards/buzzsaw/buzzsaw.c [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+
+void matrix_init_kb(void) {
+    // unused pins - C7, D4, D5, D7, E6
+    // set as input with internal pull-up enabled
+    DDRB  &= ~(1<<4 | 1<<5 | 1<<6 | 1<<7);
+    PORTB &=  (1<<4 | 1<<5 | 1<<6 | 1<<7);
+
+    DDRD  &= ~(1<<5 | 1<<6 | 1<<7);
+    PORTD &=  (1<<5 | 1<<6 | 1<<7);
```
[the remark this answers is not in the document]
Comment (germ) [reply keyboards/buzzsaw/buzzsaw.c]: They could, for some reason I use the AVR notation. If I move to ARM I'll have to change a ton in here anyway!

File: keyboards/buzzsaw/readme.md (hunk 3 above)
Comment (germ) [reply keyboards/buzzsaw/readme.md]: fixed

File: keyboards/buzzsaw/keymaps/default/rules.mk (hunk 1 above)
Comment (germ) [reply keyboards/buzzsaw/keymaps/default/rules.mk]: fixed

File: keyboards/buzzsaw/keymaps/default/rules.mk [hunk 7]
```diff
@@ -0,0 +1,23 @@
+#----------------------------------------------------------------------------
+# make gergo:germ:dfu
+# Make sure you have dfu-programmer installed!
+#----------------------------------------------------------------------------
+# Firmware options
+BALLER = no 			# Enable to ball out
+BALLSTEP = 20  			# Multiple in px to move, multiplied by layer number
+SCROLLSTEP = 1 			# Lines to scroll with ball
+MOUSEKEY_ENABLE = yes  		# Mouse keys(+4700), needed for baller
```
Comment (fauxpark) [inline keyboards/buzzsaw/keymaps/default/rules.mk]:
```suggestion
MOUSEKEY_ENABLE = yes  		# Mouse keys, needed for baller
```

File: keyboards/buzzsaw/readme.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# BuzzSaw
+
+![BuzzSaw](https://assets.bigcartel.com/product_images/247417775/IMG_20191025_1410027-01.jpeg)
+
+A Alice style Choc only keyboard that is breakable down to a 40% Split semi-ergo
+
+[More info on gboards.ca](http://gboards.ca)
+
+* Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)
```
Comment (fauxpark) [inline keyboards/buzzsaw/readme.md]:
```suggestion
* Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)
```

File: keyboards/buzzsaw/readme.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# BuzzSaw
+
+![BuzzSaw](https://assets.bigcartel.com/product_images/247417775/IMG_20191025_1410027-01.jpeg)
+
+A Alice style Choc only keyboard that is breakable down to a 40% Split semi-ergo
+
+[More info on gboards.ca](http://gboards.ca)
+
+* Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
+* Hardware Supported: Gergo (Kit, Partial, Ready)
```
Comment (fauxpark) [inline keyboards/buzzsaw/readme.md]:
```suggestion
* Hardware Supported: Gergo (Kit, Partial, Ready)
```

File: keyboards/buzzsaw/readme.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+![BuzzSaw](https://assets.bigcartel.com/product_images/247417775/IMG_20191025_1410027-01.jpeg)
+
+A Alice style Choc only keyboard that is breakable down to a 40% Split semi-ergo
+
+[More info on gboards.ca](http://gboards.ca)
+
+* Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
+* Hardware Supported: Gergo (Kit, Partial, Ready)   
+* Hardware Availability: [gboards.ca](http://gboards.ca)
```
Comment (fauxpark) [inline keyboards/buzzsaw/readme.md]:
```suggestion
* Hardware Availability: [gboards.ca](http://gboards.ca)
```

**Comment by germ** [comment]
Closing due to #8262

**Pull request closed without merging**